benchto-service/pom.xml

```diff
@@ -94,14 +99,23 @@
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-java8</artifactId>
```

Comment:
Is this needed? Can we drop Java 8 now and use 17?

Reply:
I think this was merged into `hibernate-core`  sometime around Hibernate 5.2.